BUILD.gn

```diff
@@ -74,8 +74,15 @@ group("create_dist") {
     deps += [ "build/mac:create_dist_mac" ]
   }
   if (is_linux) {
+    _linux_channel = brave_channel
+    if (brave_channel == "") {
+      _linux_channel = "stable"
+    } else if (brave_channel == "dev") {
+      _linux_channel = "unstable"
```

Comment:
other options for `brave_channel` are unchanged for linux?

Reply:
Yep, `beta` is used as-is.
`nightly` is not supported yet on linux and will be added (https://github.com/brave/brave-browser/issues/461)

Reply:
can we wrap these things together as I suggested in the other post so `brave_channel` is corrected for linux in //brave/build/config.gni ?

Reply:
Yep, this value and others mentioned in https://github.com/brave/brave-core/pull/208#discussion_r199649672, can be wrapped there.

But I'm not sure we can fetch that values from c++.

Reply:
I was thinking a public_config with defines or a generated header file?

Reply:
ok, I'll try!

Reply:
Moved to `//brave//build/config.gni`. PTAL.